cli/cmd/aeon.go

```diff
@@ -0,0 +1,96 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	aeon "github.com/tarantool/tt/cli/aeon/cmd"
+	"github.com/tarantool/tt/cli/cmdcontext"
+	"github.com/tarantool/tt/cli/modules"
+	"github.com/tarantool/tt/cli/util"
+	libconnect "github.com/tarantool/tt/lib/connect"
+)
+
+var aeonConnectCtx = aeon.ConnectCtx{
+	TransportMode: aeon.AeonTransportDefault,
+}
+
+func newAeonConnectCmd() *cobra.Command {
+	var aeonCmd = &cobra.Command{
+		Use:   "connect URI",
+		Short: "Connect to the aeon instance",
+		Long: "Connect to the aeon instance.\n\n" +
+			libconnect.EnvCredentialsHelp + "\n\n" +
+			`tt aeon connect user:pass@localhost:3013`,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			err := aeonConnectValidateArgs(cmd, args)
+			util.HandleCmdErr(cmd, err)
+			return err
+		},
+		Run: func(cmd *cobra.Command, args []string) {
+			cmdCtx.CommandName = cmd.Name()
+			err := modules.RunCmd(&cmdCtx, cmd.CommandPath(), &modulesInfo,
+				internalAeonConnect, args)
+			util.HandleCmdErr(cmd, err)
+		},
+	}
+	aeonCmd.Flags().StringVar(&aeonConnectCtx.Ssl.KeyFile, "sslkeyfile", "",
+		"path to a private SSL key file")
+	aeonCmd.Flags().StringVar(&aeonConnectCtx.Ssl.CertFile, "sslcertfile", "",
+		"path to a SSL certificate file")
+	aeonCmd.Flags().StringVar(&aeonConnectCtx.Ssl.CaFile, "sslcafile", "",
+		"path to a trusted certificate authorities (CA) file")
+
+	aeonCmd.Flags().Var(&aeonConnectCtx.TransportMode, "transport",
+		fmt.Sprintf("allowed %s", aeon.ListValidTransports()))
+	aeonCmd.RegisterFlagCompletionFunc("transport", aeonTransportCompletion)
+
+	return aeonCmd
+}
+
+func aeonTransportCompletion(cmd *cobra.Command, args []string, toComplete string) (
+	[]string, cobra.ShellCompDirective) {
+	suggest := make([]string, 0, len(aeon.AeonValidTransport))
+	for k, v := range aeon.AeonValidTransport {
+		suggest = append(suggest, string(k)+"\t"+v)
+	}
+	return suggest, cobra.ShellCompDirectiveDefault
+}
+
+// NewAeonCmd() create new aeon command.
+func NewAeonCmd() *cobra.Command {
+	var aeonCmd = &cobra.Command{
+		Use:   "aeon",
+		Short: "Manage aeon application",
+	}
+	aeonCmd.AddCommand(
+		newAeonConnectCmd(),
+	)
+	return aeonCmd
+}
+
+func aeonConnectValidateArgs(cmd *cobra.Command, args []string) error {
+	if aeonConnectCtx.TransportMode == "ssl" {
+		if !util.IsRegularFile(aeonConnectCtx.Ssl.KeyFile) {
+			return fmt.Errorf("not valid path to a private SSL key file=%q",
+				aeonConnectCtx.Ssl.KeyFile)
+		}
+		if !util.IsRegularFile(aeonConnectCtx.Ssl.CertFile) {
+			return fmt.Errorf("not valid path to an SSL certificate file=%q",
+				aeonConnectCtx.Ssl.CertFile)
+		}
+		if !util.IsRegularFile(aeonConnectCtx.Ssl.CaFile) {
+			return fmt.Errorf("not valid path to trusted certificate authorities (CA) file=%q",
+				aeonConnectCtx.Ssl.CaFile)
+		}
+	}
+	aeonConnectCtx.Username = os.Getenv(libconnect.TarantoolUsernameEnv)
+	aeonConnectCtx.Password = os.Getenv(libconnect.TarantoolPasswordEnv)
```

Comment:
We don't need the functionality at now because it is not implemented by aeon.

But if we will need the functionality that it should be ability to set login/pass with flags too.